packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -65,7 +105,7 @@ export class QuickFileOpenService implements QuickOpenModel {
                 acceptor(await Promise.all(itemPromises));
             }
         };
-        this.fileSearchService.find(lookFor, { rootUri, fuzzyMatch: true, limit: 200 }, token).then(handler);
+        this.fileSearchService.find(lookFor, { rootUri, fuzzyMatch: true, limit: 200, useGitIgnore: !this.isShowIgnored() }, token).then(handler);
```

Comment:
This line is getting really long, would you mind wrapping the object, putting one property per line?

```ts
        this.fileSearchService.find(lookFor, {
            rootUri,
            fuzzyMatch: true,
            limit: 200,
            useGitIgnore: !this.showIgnored,
        }, token).then(handler);
```

Reply:
Done